openedx/core/djangoapps/programs/models.py

```diff
@@ -65,6 +65,15 @@ class ProgramsApiConfig(ConfigurationModel):
         default=False
     )
 
+    max_retries = models.PositiveIntegerField(
+        verbose_name=_("Maximum Certification Retries"),
+        default=3,
```

Comment:
I'd make the default 11 retries. 11 retries with exponential backoff yields a maximum waiting time of 2047 seconds - about 30 minutes - as opposed to the 14 seconds you get with 3 retries. We use this more forgiving default for the ecomworker.